main.py

```
@@ -213,3 +214,16 @@ async def get_match_preview(
 ):
     result = get_match_preview_info(game_id)
     return JSONResponse(content=result)
+
+@app.get("/team_players")
+def team_players(
+    team_id: int = Query(..., description="팀 ID"),
+    user: dict = Depends(get_current_user)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)